client/packages/system/src/Immunisation/ProgramView/ProgramView.tsx

```diff
@@ -1,29 +1,78 @@
-import React, { FC, useEffect } from 'react';
+import React, { FC, useEffect, useState } from 'react';
 import {
   TableProvider,
   createTableStore,
   useBreadcrumbs,
   NothingHere,
   createQueryParamsStore,
+  DataTable,
+  useNavigate,
+  useTranslation,
+  useUrlQueryParams,
+  useColumns,
 } from '@openmsupply-client/common';
 import { Toolbar } from './Toolbar';
+import { AppBarButtons } from './AppBarButtons';
 
 // dummy data
 const data = {
-  name: 'data',
+  name: 'some program name',
 };
 
+export interface Immunisation {
+  id: string;
+  name: string;
+  vaccines: string[];
+  isNew: boolean;
+}
+
 export const ProgramComponent: FC = () => {
+  const {
+    updateSortQuery,
+    updatePaginationQuery,
+    queryParams: { sortBy, page, first, offset },
+  } = useUrlQueryParams({ filters: [{ key: 'name' }] });
+  const pagination = { page, first, offset };
+  const navigate = useNavigate();
+  const t = useTranslation('catalogue');
   const { setSuffix } = useBreadcrumbs();
 
+  const draftProgram: Record<string, Immunisation> = {};
```

Comment:
Similar to last PR, don't think we need this datastucture. I guess it will be fixed up when we get the backend...

Reply:
You will need a draft program here, as you'll want to be able to update the program name.
However this wouldn't be used in the list view as data, that will come from the vaccineCourses api request from the backend, with a filter on the current program id